vnext/ReactWindowsCore/WinRTWebSocketResource.cpp

```diff
@@ -0,0 +1,363 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+// clang-format off
```

Comment:
I think I said this somewhere else, but we shouldn't be doing this for entire files.

Reply:
it would be nice to fix clang formatting instead, but it would be a non-trivial amount of work

---
In reply to: [393492200](https://github.com/microsoft/react-native-windows/pull/4330#discussion_r393492200) [](ancestors = 393492200)

Reply:
Keeping for now, for clang-format would significantly reduce development velocity.

Reply:
if something about clang-format is slowing us down, let's fix whatever we're doing wrong instead of turning off clang-format altogether for whole files

---
In reply to: [394801482](https://github.com/microsoft/react-native-windows/pull/4330#discussion_r394801482) [](ancestors = 394801482)